server/graphql/vvm/src/queries/vvm_status_log.rs

```diff
@@ -0,0 +1,34 @@
+use async_graphql::{Context, Result};
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use service::auth::{Resource, ResourceAccessRequest};
+
+use crate::types::vvm_status_log::{VVMStatusLogConnector, VVMStatusLogResponse};
+
+pub fn get_vvm_status_log_by_stock_line_id(
+    ctx: &Context<'_>,
+    store_id: String,
+    stock_line_id: &str,
+) -> Result<VVMStatusLogResponse> {
+    let user = validate_auth(
+        ctx,
+        &ResourceAccessRequest {
+            resource: Resource::QueryPatient,
```

Comment:
Oh my, I missed a permission. Can you add a TODO here to change this and create an issue to add `View and edit vaccine vial monitor status` user permission from OG? 👀